### PR TITLE
chore(ruby): upgrade dependencies to ruby 3

### PR DESCRIPTION
**Pull request by Slashgear**
## Why

Our jekyll used Ruby on version 2, ruby 3 is out since 2020 😅 

## How

- update gemfile.lock
- add missing webrick dependency for ruby 3
- remove useless `tweet-cache` dependencies
- fix 404 in my personal avatar image
- use simpler image for the docker image

### CODE REVIEW

**Comment by Slashgear** [comment]
@pmartin When you have time, could you change the amplify deployment script for the blog in order to update the ruby version ? :)

After that we could merge this PR

File: Dockerfile [hunk 1]
```diff
@@ -1,26 +1,11 @@
-FROM ubuntu:20.04
+FROM ruby:2.7
```
Comment (pmartin) [inline Dockerfile:1]: Si on passe de Ruby 2.6 à Ruby 3 plus haut, pourquoi est-ce qu'on passe à Ruby 2.7 ici ?

Comment (Slashgear) [reply Dockerfile]: Nice catch

**Review by pmartin — APPROVED** [review]
\o/

**Pull request merged**